src/main/java/io/vertx/gsoc2023/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -75,4 +97,11 @@ private Future<RowSet<Row>> selectQuotes() {
       .query("SELECT * FROM quotes ")
       .execute());
   }
+  
+  private Future<RowSet<Row>> insertQuote(String author, String text) {
+    return pgPool.withConnection(connection -> 
+      connection
+      .preparedQuery("INSERT INTO quotes (author, text) VALUES ($1, $2) ")
```

Comment:
I had already done it as part of the commit to add /realtime endpoint. You may have been looking at the earlier commit.
However, I did miss to update the test cases before. 
Updated the tests now to use the returned response body for assertion instead of making additional get request.
Thanks for this advise.